2020-an-environment-for-sustainable-research-software-in-germany-and-beyond/manuscript.md

```diff
@@ -0,0 +1,2020 @@
+side as well as the evaluation by the reviewers requires significant
+effort. We believe that the added value significantly outweighs the
+investment but appropriate resources need to be factored in. Excellence
+as reflected in funded projects, publications, and software adoption,
+i.e. backing by a community, should be considered during selection.
+Nevertheless, we believe a good scheme should strike a balance between
+consolidating the field to few well-established software packages on one
+side and stimulating innovation and cooperation promoting diversity in
+terms of more than one monopolistic package on the other side. Last but
+not least, there is an inherent conflict between the long-term goals of
+sustainability funding a software and the necessary reevaluation to
+monitor the state of the software over time.
+
+Selection Criteria
+------------------
+
+Several evaluation schemes for research software have been proposed
+before and led to the formulation of first recommendations (Katerbow and
+Feulner 2018; Scheliga et al. 2019). Gomez-Diaz & Recio suggested the
+CDUR based on Citation, Dissemination (including aspects like license,
+web site, contact point), Use, and Research (output) (Gomez-Diaz and
+Recio 2019). Lamprecht et al. rephrased the FAIR data
+principles (Wilkinson et al. 2016) for research software (Lamprecht et
+al. 2019). Hasselbring et al. found that the adoption of FAIR principles
+is different between fields with an emphasis on reuse in computer
+science as opposed to a reproducibility focus in computational
+science (Hasselbring et al. 2019). Fehr et al. collected a set of best
+practices for the setup and publication of numerical experiments (Fehr
+et al. 2016). Jiménez et al. boiled it down to four best
+practices (Jiménez et al. 2017): public source code, community registry,
+license, and governance. Hsu et al. (Hsu, Hutchison, and Langseth 2019)
+proposed a framework of seven sustainability influences (outputs
+modified, code repository used, champion present, workforce stability,
+support from other organizations, collaboration/partnership, and
+integration with policy). They found that the various outputs are widely
+accessible, but not necessarily sustained or maintained. Projects with
+most sustainability influences often became institutionalized and met
+required needs of the community (Hsu, Hutchison, and Langseth 2019). In
+the field of open source software, the CHAOSS (Community Health
+Analytics Open Source Software) project has developed metrics to
+evaluate sustainability (CHAOSS 2019). One objective of CHAOSS is to
+automatically generate project health reports based on software that
+evaluates the metrics, with most of the metrics already covered. The UK
+Software Sustainability Institute (SSI) suggested both a subjective
+tutorial-based and a more objective criteria-based software evaluation
+scheme (Jackson, Crouch, and Baxter 2019), the latter being available as
+an online form (SSI 2019a).
```

Comment:
```suggestion
an online form (SSI 2019a). ROpenSci provides software reviews
for R developers, which have been very successful in the 
community (see https://devguide.ropensci.org/softwarereviewintro.html). 
```

It wasn't clear to me how to cite the dev guide from ROpenSci. I hope they'll anwer soon https://github.com/ropensci/dev_guide/issues/239

I understand if you do not want to include this example (you already have quite many), but I thought it might be nice to see one where they are not only talking about reviewing software but actually *doing* it.

Reply:
Another good example might be JORS. However they are less strickt and thorough than ROpenSci.